courses/free-courses-en.md

```diff
@@ -1610,6 +1610,8 @@
 ### Scala
 
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
+* [Scala & Functional Programming Essentials](https://www.udemy.com/course/rock-the-jvm-scala-for-beginners/)
```

Comment:
This course does not appear to be free when I look at it.

Reply:
@Thenlie updated the PR please can you review